src/OpenTelemetry.Exporter.Zipkin/OpenTelemetry.Exporter.Zipkin.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
-    <TargetFrameworks>net5.0;netstandard2.0;net462;</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0;net462;</TargetFrameworks>
```

Comment:
@cijothomas @alanwest @reyang @utpilla 

Was just looking at this while investigating something else. It didn't occur to me at the time, but we kind of introduced an issue doing this.

Previously we had a `net5.0` target and code like this:

```csharp
#if NET5_0_OR_GREATER
                using var response = this.httpClient.Send(request, CancellationToken.None);
#else
```

That became:

```csharp
#if NET6_0_OR_GREATER
                using var response = this.httpClient.Send(request, CancellationToken.None);
#else
```

Users on .NET 5 upgrading to latest will still compile, they will just bind to the `netstandard2.1` target. However, they have lost the optimization. So for them, upgrading was a perf hit/carried unforeseen consequences.

Reply:
Good find. Lesson is that in the future we need to align any drop of a target framework with the actual EOL date of the framework and consider which approach we're going to adopt here https://github.com/open-telemetry/opentelemetry-dotnet/issues/3448